src/util/selectors.ts

```diff
@@ -129,3 +129,15 @@ export function selectKanTildeleVeileder(
     if (!oppfolging || !tilgangTilBrukersKontor) return false;
     return oppfolging.underOppfolging && tilgangTilBrukersKontor.tilgangTilBrukersKontor;
 }
+
+export function selectTelefonnummer(personalia: Personalia | undefined): StringOrNothing {
+    const telefonEntries = personalia?.telefon ?? [];
+    const telefonPriorities = ['1', '2', '3'];
+    for (const priority of telefonPriorities) {
+        const entry = telefonEntries.find(entry => entry.prioritet === priority);
+        if (entry) {
+            return entry.telefonNr;
+        }
+    }
+    return undefined;
+}
```

Comment:
```suggestion
export function selectTelefonnummer(personalia: Personalia | undefined): StringOrNothing {
    const telefonEntries = personalia?.telefon ?? [];
    // Filtrer bort tomme telefonnumre og sorter på prioritet, velger det med lavest prioritet
    return telefonEntries
        .filter(tlfEntry => tlfEntry.telefonNr)
        .sort((a, b) => Number(a.prioritet) - Number(b.prioritet))[0]?.telefonNr;
}
```
Jeg syns Julies forslag var bedre!